src/MoBi.Presentation/Presenter/MoBiMainViewPresenter.cs

```diff
@@ -72,8 +72,6 @@ public override void Initialize()
       {
          _view.Initialize();
          updateWindowTitle();
-
-         Thread.Sleep(10000);
```

Comment:
@msevestre Added in this PR related to SVG support. Do you have any more context for it?

This is the performance profile with the sleep.
![image](https://github.com/user-attachments/assets/a8dcab1e-ad0e-4e0c-9499-e7dfe903ff1d)

There's a UI Freeze at about 29 seconds into the boot and a 10 second block with nothing happening.

I removed the sleep and the UI freeze comes in around 20 seconds and the total boot time is shorter as you'd expect.

![image](https://github.com/user-attachments/assets/bb7aa16f-59c4-493b-9bc5-63b3533f627d)